test/framework/test_subscriber.hpp

```diff
@@ -272,6 +272,63 @@ namespace framework {
       bool on_complete_called = false;
     };
 
+    /**
+     * EqualToList checks subscribed values equals to vector<T>
+     * @tparam T - observable parameter
+     */
+    template <typename T>
+    class EqualToList : public VerificationStrategy<T> {
+     public:
+      /**
+       * @param expected_values - expected vector<T> values
+       * equals to list from observable
+       */
+      explicit EqualToList(std::vector<T> expected_values) noexcept
+        : expected_values_(std::move(expected_values)) {}
+
+      explicit EqualToList(EqualToList<T> &&rhs) noexcept {
```

Comment:
What is the reason to use `explicit` keyword for move ctor?

Reply:
No reason to do this. Thank you.